core/src/global.rs

```diff
@@ -132,7 +131,7 @@ where
 	// Perform whatever tests, configuration etc are needed to determine desired context + edge size
 	// + params
 	// Hardcode to regular cuckoo for now
-	CuckooContext::<T>::new(edge_bits, proof_size, easiness_pct, max_sols)
+	CuckooContext::<T>::new(edge_bits, proof_size, EASINESS, max_sols)
```

Comment:
Sure, I gues Easiness is the only thing that's not likely to change between pows